### PR TITLE
Correct and add Vietnamese translation

### PR DESCRIPTION
**Pull request by CactiStaccingCrane**
Title is self-explanatory.

### CODE REVIEW

File: website/public/locales/vi/common.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "account_settings": "Tài khoản",
   "admin_dashboard": "Trang cho admin",
   "connect": "Liên hệ",
-  "conversational": "Chatbot AI cho tất cả mọi người",
-  "copied": "Copied",
+  "conversational": "Chatbot AI cho mọi người",
+  "copied": "Đã chép",
```
Comment (notmd) [inline website/public/locales/vi/common.json:7]:
```suggestion
  "copied": "Đã sao chép",
```

File: website/public/locales/vi/common.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
   "report_a_bug": "Báo lỗi",
   "sign_in": "Đăng nhập",
   "sign_out": "Đăng xuất",
-  "success": "Success",
+  "success": "Đã thành công",
```
Comment (notmd) [inline website/public/locales/vi/common.json:23]:
```suggestion
  "success": "Thành công",
```

**Review by notmd — CHANGES_REQUESTED** [review]

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
Will approve after @notmd's suggestions are applied

**Comment by fozziethebeat** [comment]
FYI: we had some refactoring and a conflict will also need to be addressed.

**Comment by CactiStaccingCrane** [comment]
Will make another pull request, there's a conflict and I don't know how to respond to that

**Pull request closed without merging**